components/style/properties/longhand/box.mako.rs

```diff
@@ -105,18 +107,20 @@ ${helpers.single_keyword("position", "static absolute relative fixed sticky",
             }
         }
     }
-</%helpers:single_keyword_computed>
-
-<%helpers:single_keyword_computed name="clear"
-                                  values="none left right both"
-                                  // https://drafts.csswg.org/css-logical-props/#float-clear
-                                  extra_specified="inline-start inline-end"
-                                  needs_conversion="True"
-                                  gecko_inexhaustive="True"
-                                  animation_value_type="discrete"
-                                  gecko_enum_prefix="StyleClear"
-                                  gecko_ffi_name="mBreakType"
-                                  spec="https://www.w3.org/TR/CSS2/visuren.html#flow-control">
+</%helpers:single_keyword>
+
+<%helpers:single_keyword
+    name="clear"
+    values="none left right both"
+    // https://drafts.csswg.org/css-logical-props/#float-clear
+    extra_specified="inline-start inline-end"
+    needs_conversion="True"
+    gecko_inexhaustive="True"
+    animation_value_type="discrete"
+    gecko_enum_prefix="StyleClear"
+    gecko_ffi_name="mBreakType"
+    spec="https://www.w3.org/TR/CSS2/visuren.html#flow-control"
```

Comment:
If while you're here you change this to https://drafts.csswg.org/css-box/#propdef-clear it'll make me very happy :)